playbooks/compliance_check.yaml

```
@@ -4,35 +4,33 @@
   tasks:
     - name: Run compliance script
       ansible.builtin.shell: |
-        python3 ~/Tests/scs-compliance-check.py ~/Tests/scs-compatible-iaas.yaml -s {{ cloud }} -a os_cloud={{ cloud }} -o report.yaml -C
-      register: result
+        python3 ~/Tests/cleanup.py -c {{ item }} --prefix _scs-
+        python3 ~/Tests/scs-compliance-check.py ~/Tests/scs-compatible-iaas.yaml -s {{ item }} -a os_cloud={{ item }} -o report-{{ item }}.yaml -C
+        ssh-keygen -Y sign -f ~/id_subject -n report report-{{ item }}.yaml
+        curl --data-binary @report-{{ item }}.yaml.sig --data-binary @report-{{ item }}.yaml -H "Content-Type: application/x-signed-yaml" -H "Authorization: Basic $auth" https://compliance.sovereignit.cloud/reports
       changed_when: true
       failed_when: false
       # ^^^ this task does not fail; the only failure would be if `report.yaml` didn't get produced,
       # but then the subsequent tasks would fail anyway
-
-    - ansible.builtin.debug:
-        msg: "{{ result.stdout }} {{ result.stderr }}"
-
-    - name: sign result YAML and upload to compliance monitor
-      no_log: true  # do not leak the secret in the shell part below
-      ansible.builtin.shell: |
-        ssh-keygen -Y sign -f ~/id_subject -n report report.yaml
-        curl --data-binary @report.yaml.sig --data-binary @report.yaml -H "Content-Type: application/x-signed-yaml" -H "Authorization: Basic {{ clouds_conf.zuul_ci_basic_auth }}" https://compliance.sovereignit.cloud/reports
+      environment:
+        auth: "{{ clouds_conf.zuul_ci_basic_auth }}"
```

```suggestion
        AUTH_TOKEN: "{{ clouds_conf.zuul_ci_basic_auth }}"
```

Just easier to see if this is also used like this in the `curl` call, plus convention for envvars is upper case AFAIK.

---

Sure! Somehow the examples in the docs didn't use uppercase, bu t I would prefer that as well. I have to change the curl as well, of course.

---

54e320ecb2edb0f709ef26ad059038f6b00419aa